### PR TITLE
John 04 30

### PR DESCRIPTION
**Pull request by johnvaltix**
Updates to various terraform resources, including adding VPN resources

### CODE REVIEW

File: docs/resources/profile_urlfilter.md [hunk 1]
```diff
@@ -3,23 +3,9 @@ Resource for creating and managing a URL Filter (URL) Profile
 
 ## Example Usage
 ```hcl
-resource "valtix_profile_urlfilter" "url1" {
+resource "ciscomcd_profile_urlfilter" "url1" {
```
Comment (kiranvaltix) [inline docs/resources/profile_urlfilter.md:6]: shouldnt this be valtix_profile_urlfilter?

File: docs/resources/profile_urlfilter.md [hunk 2]
```diff
@@ -217,7 +226,7 @@ default_url_filter {
 }
 ```
 
-Please check the Valtix UI (Manage -> Profiles -> URL Filtering) to obtain a list of predefined Categories.
+Please check the Valtix UI (Manage -> Profiles -> URL Filtering) to obtain a list of predefined Categories. A full list of Talos catogories is available at [Talos Intelligence Cateegories](https://www.talosintelligence.com/categories). A Talos URL loopup tool is available at [Talos Intelligence Search](https://www.talosintelligence.com/reputation_center).
```
Comment (kiranvaltix) [inline docs/resources/profile_urlfilter.md:229]: lookup instead of loopup

**Review by kiranvaltix — CHANGES_REQUESTED** [review]

File: docs/resources/profile_urlfilter.md (hunk 1 above)
Comment (johnvaltix) [reply docs/resources/profile_urlfilter.md:6]: Updated to correct

Comment (kiranvaltix) [inline docs/resources/profile_urlfilter.md:6]: still shows ciscomcd instead of valtix

**Review by kiranvaltix — APPROVED** [review]

**Pull request merged**